cdk/lib/recipes.test.ts

```diff
@@ -1,11 +1,12 @@
 import { App } from "aws-cdk-lib";
 import { Template } from "aws-cdk-lib/assertions";
+import { prodProps } from "../bin/cdk";
 import { Recipes } from "./recipes";
 
 describe("The Recipes stack", () => {
   it("matches the snapshot", () => {
     const app = new App();
-    const stack = new Recipes(app, "Recipes", { stack: "playground", stage: "TEST" });
+    const stack = new Recipes(app, "Recipes", prodProps);
```

Comment:
We now pass the actual props in here (rather than these test placeholders). 

This means that we don't need to remember to update props across 2 separate files. Hopefully this means that the snapshot remains representative of the template that we'll actually apply via CFN.